### ISSUE TITLE
Create `Management` chapter for daily spending mega case study

### ISSUE THREAD

**Issue by Bosch-0** [BitcoinDesign/Guide#582]
### TLDR: I propose we create a chapter called `Management` where we explore the design of mobile first, non-custodial, Bitcoin Lightning wallet components that do not pertain directly to onboarding and payments. 

We've had many discussions around re-structuring the guide. Part of those discussions have been around combining the `Onboarding` and `Payments` chapters into a Mega-Daily-Spending case study. I think this is a great approach, @GBKS has some good suggestions around how to do this exactly. 

To clarify, the product focus of this mega-daily-spending case study would be: **A mobile first, non-custodial, Bitcoin Lightning product**. This scope should be detailed at the beginning of the case study but that's something we can address when we get to re-structuring. 

There are many aspects of a daily spending wallet that do not pertain to either onboarding or payments, so I propose we create a chapter called `Management` where we can explore the design of these other wallet components more in-depth.

### Why is this important?

The guide needs to be concise and to the point on what it is detailing or we will end up with a jangled mess that is difficult to navigate and overly complex. Onboarding should only focus on onboarding, payments should only focus on payments. I think it is important to make this decision now while we are begging [Milestone #11](https://github.com/BitcoinDesign/Guide/milestone/12) otherwise we will be re-working what we create down the line when something like what I am proposing gets introduced into the guide.

For example, contacts pertain to payments though I think it would keep things more organized if we separated how to design contacts UI from the primary payment sections flows. Exploring it in payments would get over-complex fast. We can cross-link to a contacts page (see below) that explores the designing of contacts when that part of the payments section is covered.  

Below is a rough structure for a management Chapter:

### Accounts

- Accounts aren't widely used in Bitcoin applications ([Hexa](https://hexawallet.io/) is an example) yet but it's evident to include in any type of payments / finance app. We should be designing for the future here. [Zeus](https://zeusln.app/) is going with an account focused model in the upcoming redesign. 
- A sub-page for accounts could be `Allowances` of which @GBKS @johnsBeharry and the Alby team are exploring. 
- Could explore how users could create different accounts with different rules such as limits on daily spending or payment amounts for example. 
- @johnsBeharry has some UX work around accounting [here](https://github.com/Bosch-0/bitcoin-ux/tree/master/accounting).

### Contacts

- Contacts aren't used in Bitcoin applications yet but they seem evident to include in any type of payments / finance app. We should be designing for the future here.
- Explores how contacts UI should be designed and used. 
- @johnsBeharry has done loads of work around contacts [here](https://bitcoin-contacts.netlify.app/)

### Units & Symbols

- Move this from `Payments` to here https://github.com/BitcoinDesign/Guide/issues/576. 

### Progressive sovereignty 

- @GBKS mentioned the title `Progressive sovereignty` to me awhile ago, I really like it! 
- This could be a fun page to create or it could even be it's own chapter within the mega-case-study. The idea would be we talk about how to slowly give users more control over what they are doing - that is if they want to. Bitcoin applications, even the most beginner friendly ones, should always offer more advanced control. For example Breez is a great beginner friendly wallet BUT if you want to you can have complete control over your wallet - even interact via CLI commands in the app. I'm not saying going that far but you get the idea.
- Move the `Coin selection` to being apart of this page as a sub-page https://github.com/BitcoinDesign/Guide/issues/573. This case study is focused on Lightning but users may want to open channels with select UTXOs so I see it as still being relevant. 
- Add a `Channel management` sub-page. Covers having users manage their own channels. 
- Add a `Networking` sub-page. This would detail design flows around guiding users to setup and/or connect to their own nodes.
- Other things we could explore: Choosing different LSP's, CoinJoin, more advanced security features (lock-downs, time-locks etc.), I'm sure more things will come up as the tech stack evolves. 

@johnsBeharry opened an issue relating to this awhile ago - https://github.com/BitcoinDesign/Guide/issues/382

**Comment by Bosch-0** [comment]
Closing this as my thinking has changed around this.
